contrib/epee/include/storages/portable_storage_base.h

```diff
@@ -84,6 +84,13 @@ namespace epee
       array_entry_t():m_it(m_array.end()){}        
       array_entry_t(const array_entry_t& other):m_array(other.m_array), m_it(m_array.end()){}
 
+      array_entry_t& operator=(const array_entry_t& other)
```

Comment:
`array_entry_t& operator=(const array_entry_t&) = default` should be sufficient.

Reply:
Oh no wait, this does `m_it = m_array.end()` ... why would the copy constructor do that? Strange.

Reply:
I admit I have not taken the "deep dive" into why it uses this iterator (and why it is initialized to `end()` by default). It seems to be to allow fast iteration using `get_next_val()`, though right now you need to explicitly call `get_first_val()` first.